Cargo.toml

```diff
@@ -22,6 +9,25 @@ members = [
   "tools/build-wasm-example",
   "errors",
 ]
+[workspace.package]
+version = "0.9.0-dev"
+edition = "2021"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+
+[package]
+name = "bevy"
+version.workspace = true
+edition.workspace = true
+categories = ["game-engines", "graphics", "gui", "rendering"]
+description = "A refreshingly simple data-driven game engine and app framework"
+exclude = ["assets/", "tools/", ".github/", "crates/", "examples/wasm/assets/"]
```

Comment:
I don't think this needs to be applied to every crate. Also I'm curious how relative paths would resolve. Do they resolve to the workspace root or the package root.